### PR TITLE
Add 'not exhaustive' option to searchbar

### PR DESCRIPTION
**Pull request by klesaulnier**

### CODE REVIEW

File: src/components/search/search-bar.tsx [hunk 1]
```diff
@@ -145,31 +154,64 @@ export const SearchBar: FunctionComponent<SearchBarProps> = ({ inputRef }) => {
...
+                ...other
+            }: {
+                children: React.ReactNode;
+            }) => {
+                return (
+                    <Paper {...other}>
+                        <Typography
+                            variant="body1"
+                            style={{
+                                color: theme.palette.info.main,
```
Comment (AAJELLAL) [inline src/components/search/search-bar.tsx]: Proposition: Move the style definition to the top of the class.

Comment (jamal-khey) [reply src/components/search/search-bar.tsx]: i don't know it is possible to do in this case, because i am using the theme that comes from a custum hook

**Review by klesaulnier — COMMENTED** [review]
Approved

File: src/components/search/search-bar.tsx (hunk 1 above)
Comment (jamal-khey) [reply src/components/search/search-bar.tsx]: fixed by Kevin

**Review by AAJELLAL — APPROVED** [review]
Tests and code are OK.
No console errors

**Review by jamal-khey — APPROVED** [review]

**Pull request merged**